config/config.go

```diff
@@ -80,6 +82,28 @@ func LoadGenesisConfig(filename string) (*GenesisConfig, error) {
 	return &nc, nil
 }
 
+func (gc *GenesisConfig) ComputeGenesisHash() (types.Hash, error) {
+	hasher := ktypes.NewHasher()
+
+	hasher.Write([]byte(gc.ChainID))
+	hasher.Write(gc.Leader)
+
+	// sort the validators by public key
+	sort.Sort(ktypes.ByPubKey(gc.Validators))
+	for _, v := range gc.Validators {
+		hasher.Write(v.PubKey)
+		binary.Write(hasher, binary.BigEndian, v.Power)
+	}
+
+	binary.Write(hasher, binary.BigEndian, gc.MaxBlockSize)
+	binary.Write(hasher, binary.BigEndian, gc.JoinExpiry)
+	binary.Write(hasher, binary.BigEndian, gc.VoteExpiry)
+	binary.Write(hasher, binary.BigEndian, gc.DisabledGasCosts)
+	binary.Write(hasher, binary.BigEndian, gc.MaxVotesPerTx)
+
+	return hasher.Sum(nil), nil
```

Comment:
Oh, I think we can have no error return? Unless we think there's a need to check for errors from binary.Write and I don't see one.